tests/.env

```diff
@@ -7,4 +7,4 @@ ARTIFACTS_URL=http://artifacts
 GITHUB_API_URL=http://fake-github-url:5050
 GITHUB_API_COMPANY=scality
 GITHUB_API_ENABLED=true
-GITHUB_USER_ALLOWED_UPLOAD="username-pass"
```

Comment:
This one was really tricky ! The " chars were part of the value. So this made the tests script fail.